openslides/core/static/js/core/site.js

```diff
@@ -343,6 +343,13 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+.config([
+    '$sessionStorageProvider',
+    function ($sessionStorageProvider) {
+        $sessionStorageProvider.setKeyPrefix('OS-');
```

Comment:
Ok, should the `csrftoken` also be renamed to `OpenSlidesCsrfToken` to keep every cookie and storage with the same prefix?